tests/integration/test_complete_run.cfg

```diff
@@ -83,18 +84,18 @@ years = "1850:1854:2", "1850:1854:4",
   [[ atm_monthly_180x360_aave ]]
   climo_diurnal_frequency = "diurnal_8xdaily"
   climo_diurnal_subsection = "atm_monthly_diurnal_8xdaily_180x360_aave"
-  dc_obs_climo = '/lcrc/group/e3sm/public_html/e3sm_diags_test_data/unit_test_complete_run/obs/climatology'
   sets = "lat_lon","zonal_mean_xy","zonal_mean_2d","polar","cosp_histogram","meridional_mean_2d","enso_diags","qbo","diurnal_cycle","annual_cycle_zonal_mean","streamflow", "zonal_mean_2d_stratosphere",
-  streamflow_obs_ts = '/lcrc/group/e3sm/public_html/e3sm_diags_test_data/unit_test_complete_run/obs/time-series/'
 
   [[ atm_monthly_180x360_aave_environment_commands ]]
-  environment_commands="source /home/ac.forsyth2/miniconda3/etc/profile.d/conda.sh; conda activate e3sm_diags_dev_20220228"
+  # To run this test, replace conda environment with your e3sm_diags dev environment
+  environment_commands="source /home/$USER/miniconda3/etc/profile.d/conda.sh; conda activate e3sm_diags_dev_20220228"
   sets = "qbo",
   ts_subsection = "atm_monthly_180x360_aave"
 
   [[ atm_monthly_180x360_aave_tc_analysis ]]
   # Running as its own subtask because tc_analysis requires jobs to run sequentially, which slows down testing
   sets = "tc_analysis",
+  # TODO: The data from this path doesn't appear to be in /lcrc/group/e3sm/diagnostics/observations/Atm
```

Comment:
@chengzhuzhang The data from this path doesn't appear to be in `/lcrc/group/e3sm/diagnostics/observations/Atm`. What is the proper path for the TC obs data?

Reply:
There is a directory structure under /Atm. i.e. https://web.lcrc.anl.gov/public/e3sm/diagnostics/observations/Atm/tc-analysis/

Reply:
Same as other comment. I see the TC-analysis in `/lcrc/group/e3sm/public_html/diagnostics/observations/Atm/tc-analysis` but the `base_path` from Mache is `/lcrc/group/e3sm/public_html/diagnostic_output`, not `/lcrc/group/e3sm/public_html/diagnostics/`. 

In this case however, since we have the `tc_obs` parameter, we can set it to be something different. That said, I think the better option would be to move the TC data to the Mache path.